Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Set the base image
+FROM openjdk:8-jre-alpine
+
+# Dockerfile author / maintainer
+MAINTAINER SourceLag.org <stephen.powis@gmail.com>
+
+# This copies binary distribution inside the image
+RUN mkdir -p /app
+ADD kafka-webview-ui/target/kafka-webview-ui-1.0.0-bin.zip /app
+
+# Extract the distribution
+WORKDIR /app
+RUN unzip -x kafka-webview-ui-*-bin.zip
+RUN rm -f kafka-webview-ui-*-bin.zip
```

Comment:
run as one step to minimise the number of docker layers and reduce the final size of your image
```
RUN unzip -x kafka-webview-ui-*-bin.zip && \
        rm -f kafka-webview-ui-*-bin.zip
```